java/docs/api/class-elementhandle.mdx

```diff
@@ -1075,12 +1075,12 @@ This method does not work across navigations, use [Page.waitForSelector()](/api/
 [JSHandle]: /api/class-jshandle.mdx "JSHandle"
 [Keyboard]: /api/class-keyboard.mdx "Keyboard"
 [Locator]: /api/class-locator.mdx "Locator"
-[LocatorAssertions]: /test-assertions.mdx "LocatorAssertions"
+[LocatorAssertions]: /api/class-locatorassertions.mdx "LocatorAssertions"
```

Comment:
Is this an intended change? I don't see such an mdx file in this PR.

Reply:
Intended, but yeah, files are missing.